src/lib.rs

```diff
@@ -181,6 +194,11 @@ pub fn probe() -> ProbeResult {
             "CARootCertificates.pem",
             "tls-ca-bundle.pem",
         ];
+        #[cfg(target_os = "freebsd")]
```

Comment:
Do you actually need these? Why?

Reply:
Not necessarily, it just reduces the list because the other files will never exist by default with the other dirs I have supplied. `"ca-root-nss.crt"` is missing. I can it and collapse the list.

Sake of completeness.

Reply:
@djc Is your concern resolved?

Reply:
@ctz @cpu thoughts? I'm a bit wary of increasing future support requirements from future FreeBSD users if we limit this. On the other hand, maybe it's fine to try and get away with the shorter list.